<launch>
  <param name="/Brain5Camera/vrpn_server_ip" value="192.168.1.103"/>
  <!-- ros_vrpn_client -->
  <node pkg="ros_vrpn_client" type="ros_vrpn_client" name="$(optenv MYSELF Brain5Camera)" required="true"/>

  <!--stereo_image_proc-->  
  <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" ns="Brain5" required="true">
        <param name="approximate_sync" type="bool" value="true"/>
  </node>
  
  <!-- pgrcamera_driver -->
  <param name="/basic_camera_driver/left_cam_url" value="file://$(find pgrcamera_driver)/include/left_Brain5_oct8.yaml" /> 
  <param name="/basic_camera_driver/right_cam_url" value="file://$(find pgrcamera_driver)/include/right_Brain5_oct8.yaml"/> 
  <param name="/robot/name" value="Brain5"/>
  <node pkg="pgrcamera_driver" type="basic_camera_driver" name="basic_camera_driver" required="true"/>

  <!-- image_acquisition -->
  <node pkg="single_localization" type="ima_acquisition" name="ima_acquisition" required="true" output="screen"/>

</launch>
